qiskit/transpiler/passmanager.py

```diff
@@ -349,21 +349,22 @@ class StagedPassManager(PassManager):
     pass manager you are not able to modify the individual passes and are only able
     to modify stages.
 
-    By default instances of ``StagedPassManager`` define a typical full compilation
+    By default, instances of ``StagedPassManager`` define a typical full compilation
     pipeline from an abstract virtual circuit to one that is optimized and
     capable of running on the specified backend. The default pre-defined stages are:
 
-    #. ``init`` - any initial passes that are run before we start embedding the circuit to the backend
-    #. ``layout`` - This stage runs layout and maps the virtual qubits in the
-       circuit to the physical qubits on a backend
-    #. ``routing`` - This stage runs after a layout has been run and will insert any
-       necessary gates to move the qubit states around until it can be run on
-       backend's coupling map.
-    #. ``translation`` - Perform the basis gate translation, in other words translate the gates
-       in the circuit to the target backend's basis set
-    #. ``optimization`` - The main optimization loop, this will typically run in a loop trying to
-       optimize the circuit until a condition (such as fixed depth) is reached.
-    #. ``scheduling`` - Any hardware aware scheduling passes
+    #. ``init`` - Initial passes to run before embedding the circuit to the backend.
+    #. ``layout`` - Maps the virtual qubits in the circuit to the physical qubits on
+       the backend.
+    #. ``routing`` - Inserts swap gates as needed to move the qubit states around until
+       the circuit can be run with the chosen layout on the backend's coupling map.
+    #. ``translation`` - Translates the gates in the circuit to the target backend's
+       basis gate set.
+    #. ``optimization`` - Optimizes the circuit to reduce the cost of executing it.
+       These will typically run in a loop until a condition (such as a maximum depth)
```

Comment:
What do you mean by maximum depth in this case?

Reply:
I thought that "maximum" was better than "fixed" because the resulting depth might not be a specific number, but could be lower.

Reply:
Well, the "fixed" used here before was referring to a fixed depth between to iterations of the optimization loop. Specifically the logic for the preset optimization loops in level 1 and 2 is something like:

```python
old_depth = 0
new_depth = qc.depth()
while not old_depth == new_depth:
    old_depth = qc.depth()
    qc = run_optimization_passes()
    new_depth = qc.depth()
```
(it does this for size and depth though). I can see how the use of fixed here is confusing though. My issue around using "maximum" though is I think of the optimization loop as trying to minimize the depth of the circuit.

Reply:
I see. Indeed, I misunderstood the original wording. I thought it meant there was a threshold for a maximum allowed depth. I've rewritten it as follows:
```
    #. ``optimization`` - Optimizes the circuit to reduce the cost of executing it.
       These passes will typically run in a loop until a convergence criteria is met.
       For example, the convergence criteria might be that the circuit depth does not
       decrease in successive iterations.
```